tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -105,20 +105,20 @@ Feature: spaces management
       | name   | id     |
       | team A | team.a |
     And "Admin" adds the following members to the space "team A" using API
-      | user  | role   | shareType |
-      | Brian | editor | space     |
-      | Carol | viewer | space     |
-      | David | viewer | space     |
-      | Edith | viewer | space     |
+      | user  | role     | shareType |
+      | Brian | Can edit | space     |
+      | Carol | Can view | space     |
+      | David | Can view | space     |
+      | Edith | Can view | space     |
```

Comment:
CC @saw-jan @SwikritiT are we giving the exactly same role that is in the display text of UI for different role from the feature file it self or should we substitute it inside the implementation and keep the feature file the same ?

Reply:
I really don't know :thinking: . Maybe the previous one was better.
But let @ScharfViktor have thoughts on it

Reply:
@SagarGi  role name is changed only in the WebUI part. Inside implementation is the same as before on the web

Reply:
Valid point :+1: 
Since it's just the refactoring kind of PR. Let's wait for @ScharfViktor what he has to say.
What do you think?

Reply:
We know that `editor = can edit`. 
In my opinion, it is better to do the same in web and test. LGTM